larq/testing_utils.py

```diff
@@ -150,15 +149,13 @@ def layer_test(
     # See b/120160788 for more details. This should be mitigated after 2.0.
     model = tf.keras.models.Model(x, layer(x))
     if _thread_local_data.run_eagerly is not None:
-        # TensorFlow < 1.14 + eager is broken, skip the test
```

Comment:
This entire function is just [forked from Tensorflow](https://github.com/tensorflow/tensorflow/blob/3400bf99b9cda9603db18680435091f595c4f6c5/tensorflow/python/keras/testing_utils.py#L75) which _might_ be due to TF 1.13 incompatibility. We can check if it makes sense to remove it in a future PR. Although this might conflict with #314